client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/connected/GetSecurityHotspotRequestParams.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.client.api.connected;
+
+public class GetSecurityHotspotRequestParams {
+  public final String hotspotKey;
+  public final String projectKey;
+
+  public GetSecurityHotspotRequestParams(String hotspotKey, String projectKey) {
```

Comment:
What about branch/pr? Is it not required to get the hotspot?

Reply:
It is not required, indeed. Only the hotspot "key" is needed